src/main/java/org/codehaus/mojo/flatten/FlattenMojo.java

```diff
@@ -550,6 +542,28 @@ protected Model createFlattenedPom( File pomFile ) throws MojoExecutionException
         ModelBuildingRequest buildingRequest = createModelBuildingRequest( pomFile );
         Model effectivePom = createEffectivePom( buildingRequest, isEmbedBuildProfileDependencies(), this.flattenMode );
 
+        // parse dependency version in <profile>
+        Dependencies managedDependencies = new Dependencies();
+        if (effectivePom.getDependencyManagement() != null
+                && effectivePom.getDependencyManagement().getDependencies() != null)
+            managedDependencies.addAll(effectivePom.getDependencyManagement().getDependencies());
+        if (effectivePom.getProfiles() != null) {
+            for (Profile prof : effectivePom.getProfiles()) {
+                if (prof.getDependencies() == null)
+                    continue;
```

Comment:
continue wrap in if statement rather than continue.

Reply:
update: use curly brackets instead of continue